src/main/java/logging/Person.java

```diff
@@ -0,0 +1,115 @@
+package logging;
+
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.net.MalformedURLException;
+
+public class Person {
+    private String name;
+    private String surname;
+    private String email;
+    private ChildCount childCount;
+    private Pet pet;
+
+    private static Logger logger;
+
+    enum ChildCount {
+        One,
```

Comment:
enum values are constants and they must follow convention UPPER_CASE_UNDERSCORE

Reply:
Done